programs/util.h

```diff
@@ -127,6 +127,8 @@ extern int g_utilDisplayLevel;
     typedef struct stat stat_t;
 #endif
 
+int g_excludeCompressedFiles;
+static const char compressedFileExtensions[4][10] = {".zst",".gz",".xz",".lz4"};
```

Comment:
We may need to `typedef` a type here,
in order to make sure that all invocations of this type use the same constants,
and also to make it easier to evolve this list in the future.

Reply:
Also,
we already have a list of extensions, but in `fileio.h` :
https://github.com/facebook/zstd/blob/dev/programs/fileio.h#L33

I'm wondering if it would make sense to join these lists, so that there is a single point of definition.

Reply:
Indeed. Also note that my PR #1848 creates an actual null-terminated list of extensions. It is better to use that pattern (or alternatively a `sizeof(array)/sizeof(array[0])` idiom) rather than rely on a hard-coded length at every use.